src/bindings/python/src/openvino/runtime/opset13/ops.py

```diff
@@ -137,7 +137,7 @@ def fake_convert(
                   of the same type as the data, and shape Numpy-broadcastable to data.
     :param shift: Optional tensor with value to subtract before and add after conversion of the data input value,
                   of the same type as the data, and shape Numpy-broadcastable to data.
-    :param destination_type: Type to emulate, string of either "f8e4m3" or "f8e5m2".
+    :param destination_type: Type to emulate, string of either "f8e4m3", "f8e5m2" or "f8e8m0".
```

Comment:
Is not required, fake convert has specified two types for support

Reply:
Done